django/core/api_internes/paginators.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import pagination
+from rest_framework.response import Response
+
+
+class DocurbaPagination(pagination.PageNumberPagination):
+    # https://github.com/encode/django-rest-framework/blob/main/rest_framework/pagination.py#L230-L236
+    def get_paginated_response(self, data: dict) -> Response:
+        response = super().get_paginated_response(data)
+        # https://docs.djangoproject.com/en/6.0/ref/paginator/#django.core.paginator.Paginator.num_pages
+        response.data["num_pages"] = self.page.paginator.num_pages
```

Comment:
Ce paginator me semble superflu :
- Comme indiqué dans un autre commentaire, je pense que nous n'avons pas besoin de pagination
- Si on la garde, le code JS n'a pas besoin de `num_pages` pour itérer, il peut utiliser la présence ou non d'une URL dans le champ `next`.